webapi/webapi-contactsmanager-w3c-tests/contactsmanager/ContactsManager_save_Contact_fullset.html

```diff
@@ -68,7 +68,26 @@
     preferred: true,
     value: 'msn:kangx.xu@intel.com'
   });
-  var data = new Date(1987,10,27);
+  Date.prototype.format = function(format) {
```

Comment:
Yeah,I made reference from this website http://blog.csdn.net/chinet_bridge/article/details/15026729